documentation/configure.markdown

```diff
@@ -180,6 +180,8 @@ When this is set to true, debugging information will be written to `<vardir>/deb
 
 ### `url-prefix`
 
+> **Deprecated:** This setting will be removed in the future and url-prefixes should be configured using the `web-router-service`.
```

Comment:
I think we need to document the web-router-service, I don't think we do that currently